core/src/main/java/com/orientechnologies/orient/core/storage/cache/local/twoq/O2QCache.java

```diff
@@ -380,7 +381,7 @@ private UpdateCacheResult doLoad(long fileId, long pageIndex, boolean checkPinne
   }
 
   @Override
-  public OCacheEntry allocateNewPage(long fileId, OWriteCache writeCache) throws IOException {
+  public OCacheEntry allocateNewPage(long fileId, OWriteCache writeCache, boolean verifyChecksums) throws IOException {
```

Comment:
If there is an implicit assertion that this may never happens, it's better to insert an explicit check for such assertion in `OWOWCache#cacheFileContent` and remove `verifyChecksums` from `allocateNewPage `.